src/m365/spo/commands/file/FileProperties.ts

```diff
@@ -21,4 +21,5 @@ export interface FileProperties {
   UIVersion: number;
   UIVersionLabel: string;
   UniqueId: string;
+  RoleAssignments?: any
```

Comment:
similar comment to #4063 to avoid using `any` 
Maybe we could resolve this in the `web get` PR and when pushed just align the other PRs. Alternatively, we could to the same in all PRs and I will resolve the conflicts when merging.  The approach is as you prefer 😉

Reply:
Thx Adam! I've put it in this pr so it won't get broken